includes/Modules/Idea_Hub.php

```diff
@@ -128,6 +159,96 @@ function( $maybe_empty, $postarr ) {
 		$this->post_topic_setting->register();
 	}
 
+	/**
+	 * Shows admin notification for idea hub ideas on post list screen.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $notices Array of admin notices.
+	 *
+	 * @return array Array of admin notices.
+	 */
+	private function googlesitekit_admin_notices( $notices ) {
+		if ( 'edit-post' !== get_current_screen()->id ) {
```

Comment:
Same here, we need to make sure that the current post type is `post` (`$GLOBALS['typenow'] === 'post'`)

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/918df4e4ad848360545de51b12ac4167db594c2b